app_legacy/Helpers/render/game.php

```diff
@@ -81,7 +81,7 @@ function renderGameTitle(?string $title = null, bool $tags = true): string
             . "<span class='tag-arrow'></span>"
             . "<span>$subset</span>"
             . "</span>";
-        $updateHtml($html, $matches[0], $tags ? " $span" : '');
+        $updateHtml($html, " $matches[0]", $tags ? " $span" : '');
```

Comment:
shouldn't the space be part of the match? what if the name doesn't have a preceding space?

(something maybe like this [untested])
`/\[Subset - (.+)\]/` => `/\s?\[Subset - (.+)\]/`